packages/build/src/merge-mocha-configs.js

```diff
@@ -0,0 +1,53 @@
+// Copyright IBM Corp. 2017,2020. All Rights Reserved.
+// Node module: @loopback/build
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const debug = require('debug')('loopback:build:merge-mocha-configs');
+
+module.exports = mergeMochaConfigs;
+
+/**
+ * Merge multiple Mocha configuration files into a single one.
+ *
+ * @param {MochaConfig[]} configs A list of Mocha configuration objects
+ * as provided by `.mocharc.js` files.
+ */
+function mergeMochaConfigs(...configs) {
```

Comment:
Can we use https://lodash.com/docs/4.17.15#mergeWith instead?

Reply:
IDK, do we want to introduce dependency on `lodash`? Personally, I try to keep our dependencies small.